src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -89,6 +89,16 @@ void POWERMGNT::setPower(PowerLevels_e Power)
             Radio.SetOutputPower(0b1111); //15
             CurrentPower = PWR_50mW;
         }
+        if (Power == PWR_100mW
```

Comment:
Is this an extra guard in case power is somehow set beyond MaxPower?

Reply:
Oh, I see why below 👍

Reply:
It's so that if you ask for anything higher than 50mw, it will set it to max power. Before this change it did nothing (didnt change power) if you gave it anything higher than 50mw.

Reply:
It seems this code is unreachable because of the condition on line 75